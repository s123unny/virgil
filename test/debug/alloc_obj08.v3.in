b ../core/alloc_obj08.v3 5
run
bt
info l
c
q
