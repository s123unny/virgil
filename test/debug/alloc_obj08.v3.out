breakpoint|../core/alloc_obj08.v3 5|0
stopOnBreakpoint
bt|main|../core/alloc_obj08.v3|5
variable|0|a|0|int|false
variable|1|b|0|int|false
variable|2|x|#1:alloc_obj08|alloc_obj08|true
end
result|0
